<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>课程管理工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QTableWidget" name="tableWidgetTimetable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="verticalFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelHello">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>欢迎，清水无音pary</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="horizontalFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelTimeHint">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>当前时间：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>第X周，周〇，HH:MM</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelInfoHint">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>课程信息：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetInfo">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <widget class="QMenu" name="menu_load">
     <property name="title">
      <string>载入文件(&amp;L)</string>
     </property>
     <addaction name="action_info"/>
     <addaction name="action_time"/>
    </widget>
    <addaction name="menu_load"/>
    <addaction name="action_reload"/>
   </widget>
   <widget class="QMenu" name="menu_setting">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="action_date"/>
    <addaction name="action_welcome"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_setting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_info">
   <property name="text">
    <string>载入课程信息(&amp;I)...</string>
   </property>
  </action>
  <action name="action_time">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>载入时间表(&amp;T)...</string>
   </property>
  </action>
  <action name="action_reload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>重新载入所有文件(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_date">
   <property name="text">
    <string>设置学期起始日(&amp;D)...</string>
   </property>
  </action>
  <action name="action_welcome">
   <property name="text">
    <string>设置欢迎语(&amp;W)...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
